packages/widget-react/src/pages/tx/MsgExecuteArgs.tsx

```
@@ -0,0 +1,82 @@
+      <div className={styles.key}>
+        args
+        <Button.Small onClick={() => setDecoded((d) => !d)} disabled={!isDecodeAvailable}>
+          {decoded ? "Encode" : "Decode"}
+        </Button.Small>
+      </div>
+      {decoded ? (
+        <div className={styles.decoded}>
+          {!!functionParams &&
+            stringifyBcsArgs(msg["args"], functionParams).map(
+              ({ type, decoded, encoded, error }) => (
+                <div className={styles.arg}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)